classes/migration/upgrade/v3_4_0/I10962_UpdateEmailTemplateVariables.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @file classes/migration/upgrade/v3_4_0/I10962_UpdateEmailTemplateVariables.php
+ *
+ * Copyright (c) 2014-2025 Simon Fraser University
+ * Copyright (c) 2000-2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class I10962_UpdateEmailTemplateVariables
+ *
+ * @brief Remap {$journalAcronym} to {$contextAcronym} in the COPYEDIT_REQUEST email template.
+ */
+
+namespace PKP\migration\upgrade\v3_4_0;
+
+use Illuminate\Support\Facades\DB;
+use PKP\install\DowngradeNotSupportedException;
+use PKP\migration\Migration;
+
+class I10962_UpdateEmailTemplateVariables extends Migration
+{
+    public function up(): void
+    {
+        DB::transaction(function () {
+            $map = [
+                'journalAcronym' => 'contextAcronym',
+                'journalName' => 'contextName',
+                'journalUrl' => 'contextUrl',
+                'journalSignature' => 'contextSignature',
+            ];
```

Comment:
If I understand correctly, this is only considering the case of `OJS` but not `OMP/OPS` where we have like `pressName` or `serverName` , e.g specific to OMP/OPS . I think this is good but this core/shared lib class should have only the core migration define and abstract method for variable mapping so that each app can have it's own variable modifications . As by updating the `classes/migration/upgrade/v3_4_0/InstallEmailTemplates.php`, it will have effect when updating form 3.3.4-x but for installation upgrading form 3.5.0-x (already upgraded to 3.5.0), need to accommodate that also . Please correct me if I am mistaken .